keras/backend/theano_backend.py

```diff
@@ -642,6 +700,67 @@ def conv2d(x, kernel, strides=(1, 1), border_mode='valid', dim_ordering='th',
     return conv_out
 
 
+def conv3d(x, kernel, strides=(1, 1, 1), border_mode='valid', dim_ordering='th',
+           image_shape=None, filter_shape=None):
+    '''
+    Run on cuDNN if available.
```

Comment:
I think this is because nnet.conv2d is newly introduced by theano.
> As of December 2015, a new conv2d interface has been introduced. nnet.conv2d defines an 
>...
>By default, if cuDNN is available, we will use it, otherwise we will fall back to using the gemm version (slower then cuDNN in most cases and uses more memory). 
> ...
> Old conv2d interface is still accessible through nnet.conv.conv2d.

In conv2d of theano, it still using nnet.conv.conv2d, which is the old interface.
I think you are right, use nnet.conv2d should be better, maybe conv2d should be updated as well, but that should be done in another.